identity/src/main/java/com/stripe/android/identity/IdentityActivity.kt

```diff
@@ -127,31 +129,31 @@ internal class IdentityActivity :
                 onSuccess = {
                     finishWithResult(
                         if (it.submitted) {
-                            identityViewModel.sendAnalyticsRequest(
+                            lifecycleScope.launch {
```

Comment:
suggestion - maybe making a component that fires events using `GlobalScope` will reduce coroutine boilerplate here? this is [how we do in connections](https://github.com/stripe/stripe-android/blob/d5a78f7d6587c5996f81b7611c967d08554c9e2b/financial-connections/src/main/java/com/stripe/android/financialconnections/analytics/FinancialConnectionsAnalyticsTracker.kt#L99):